tools/istio-iptables/pkg/builder/iptables_builder_impl.go

```diff
@@ -86,57 +87,84 @@ func (rb *IptablesBuilderImpl) AppendRuleV6(chain string, table string, params .
 	return rb
 }
 
-func (rb *IptablesBuilderImpl) BuildV4() [][]string {
-	rules := [][]string{}
+func (rb *IptablesBuilderImpl) buildRules(isIpv4 bool, rules []*Rule) [][]string {
+	output := [][]string{}
+	command := dependencies.IPTABLES
```

Comment:
I would just pass the command directly as a parameter instead of the boolean, since this is the only thing that the boolean controls.

Reply:
done